data-model.sql

```diff
@@ -164,20 +153,21 @@ CREATE TABLE audit_request_source (
 	user_name varchar(100),
 	user_is_requestor boolean default true,
 	role_id_code varchar(100), -- EV(110153, DCM, "Source")
-	network_access_point_type_code smallint, --"1" for machine (DNS) name, "2" for IP address
+	network_access_point_type_code varchar(1), --"1" for machine (DNS) name, "2" for IP address
```

Comment:
If this is a code-based lookup, it probably should be a foreign-key relationship to a table of those codes